manifests/o/Oracle/VirtualBox/7.0.2/Oracle.VirtualBox.locale.en-US.yaml

```diff
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Oracle.VirtualBox
+PackageVersion: 7.0.2
+PackageLocale: en-US
+Publisher: Oracle Corporation
+PublisherUrl: https://www.oracle.com
+PublisherSupportUrl: https://www.oracle.com/virtualization/virtualbox
+PrivacyUrl: https://www.oracle.com/legal/privacy
+Author: Oracle Corporation
+PackageName: Oracle VM VirtualBox
+PackageUrl: https://www.virtualbox.org
+License: GPL-3.0-only
+LicenseUrl: https://www.virtualbox.org/wiki/Licensing_FAQ
+Copyright: Copyright © 1995, 2022, Oracle and/or its affiliates.
+CopyrightUrl: https://www.oracle.com/legal/terms.html
+ShortDescription: VirtualBox is a general-purpose full virtualizer for x86 hardware, targeted at server, desktop and embedded use.
+Description: |-
+  VirtualBox is a powerful x86 and AMD64/Intel64 virtualization product for enterprise as well as home use.
+  Not only is VirtualBox an extremely feature rich, high performance product for enterprise customers, it is also the only professional solution that is freely available as Open Source Software under the terms of the GNU General Public License (GPL) version 2.
```

Comment:
License was fixed above (GPL-3.0-only). But the description here still refers to GPL version 2.